test_redirects/map_globalconf.py

```diff
@@ -213,8 +213,8 @@
     url_test('/firefox/mobile/faq/?os=firefox-os', '/firefox/os/faq/'),
 
     # Bug 986174
-    url_test('/firefox/mobile/features/', '/firefox/android/'),
-    url_test('/firefox/mobile/faq/', '/firefox/android/faq/'),
+    url_test('/{m,{firefox/,}mobile}/features/', '/firefox/android/'),
+    url_test('/{m,{firefox/,}mobile}/faq/', '/firefox/android/faq/'),
```

Comment:
nm, I think I've got it. If I'm reading this correctly now, 216 should expand to:
`/m/features/`
`/firefox/mobile/features/`
`/mobile/features/`
and 217 to:
`/m/faq/`
`/firefox/mobile/faq/`
`/mobile/faq/`
which lgtm.